build.sbt

```diff
@@ -365,10 +365,11 @@ lazy val servlet = libraryProject("servlet")
     libraryDependencies ++= Seq(
       javaxServletApi % Provided,
       jettyServer % Test,
-      jettyServlet % Test,
+      jettyServlet % Test
     ),
   )
   .dependsOn(server % "compile;test->test")
+  .dependsOn(asyncHttpClient % Test)
```

Comment:
This dependency is going to be a pain as we test across repos after the Great Schism.  We get close to circular dependencies when we test our servers with our clients.  How hard would it be to do this with raw AHC?

Reply:
I will do it.

Reply:
Done.